README.md

```diff
@@ -1,5 +1,5 @@
 # [OpenVINO™ Toolkit](https://01.org/openvinotoolkit) - Open Model Zoo repository
-[![Stable release](https://img.shields.io/badge/version-2021.4-green.svg)](https://github.com/openvinotoolkit/open_model_zoo/releases/tag/2021.4)
```

Comment:
This link does not refer to a _file_ in the repository, so it should not be changed. Links to files have `/blob/` after the repository name.

There could also be links to directories (although I don't think the documentation contains any at the moment). They have `/tree/` in the URL and should be handled the same way as links to files.